### ISSUE TITLE
Failing to log in using Firefox

### ISSUE THREAD

**Issue by magthe** [kryptco/kr-u2f#48]
Since a few days I've been unable to log in using Firefox (https://twitter.com/magthe/status/1282555307919585281?s=20).

### Computer
OS: Linux (ArchLinux, 5.7.8-arch1-1)
Browser: Firefox 78.0.2 (64-bit) (I've tried 78.0 and 78.0.1 too)
Add-on version: 1.0.18

### Mobile
Android: 10
Make and model: Mi A2
App version: 2.5.5

### Behaviour
When trying to log in the phone app says I'm logged in but the site never moves past the login page.

I see the following in the browsers console:

```
wrap failed with error: EvalError: call to eval() blocked by CSP content_script.js:14166:17
    injectU2fInterface moz-extension://afa8a50f-8845-4db0-9c46-6ee4fd7c0dfa/js/content_script.js:14166
    679 moz-extension://afa8a50f-8845-4db0-9c46-6ee4fd7c0dfa/js/content_script.js:13877
    __webpack_require__ moz-extension://afa8a50f-8845-4db0-9c46-6ee4fd7c0dfa/js/content_script.js:20
    <anonymous> moz-extension://afa8a50f-8845-4db0-9c46-6ee4fd7c0dfa/js/content_script.js:63
    <anonymous> moz-extension://afa8a50f-8845-4db0-9c46-6ee4fd7c0dfa/js/content_script.js:66
Content Security Policy: The page's settings blocked the loading of a resource at eval ("script-src").
```

**Comment by agrinman** [comment]
I think we've isolated this issue. Working on a fix for this now.
Thanks for reporting it.

**Comment by agrinman** [comment]
Also reported as an issue for safari on GitHub...will also work to address this.

**Comment by Danimal4326** [comment]
Also seeing this on 
Firefox 79.0
macOS 10.15.5
IOS 13.6

**Comment by magthe** [comment]
It's been quiet on this issue. I just tried again with Firefox 81 and I still see this issue.

**Comment by tkw1536** [comment]
Similar, I have actually been forced to not use krypton for login because of this.

**Comment by magthe** [comment]
> Similar, I have actually been forced to not use krypton for login because of this.

Same here, but I've not managed to find something quite as convenient. The one I found, that suits me the best is `oathtool` :disappointed: 

Is there someone else making something like Krypton?

**Comment by systemcrash** [comment]
Cannot log in, tried removing, and adding once again - Krypton on iOS says OK! But Github immediately says: `⚠️  Security key registration failed. Try again`. One time codes in Krypton work, however. But not all github actions ask for a code when it thinks you have a key....

FB also seems to get stuck. 

All started after a restore from backup I did on macOS yesterday.

**Comment by michaelblyons** [comment]
There has been zero activity from the organization since [their acquisition](https://krypt.co/blog/announcements/kryptco-is-now-part-of-akamai.html) by Akamai. Don't hold your breath for an update. It's just another story of a promising start-up being bought up and canceled by a big internet player.

> More updates coming soon.
>
> — _Alex Grinman, Co-founder krypt.co_

That's what you thought, Alex, but that's not how it works in the Game.

**Comment by magthe** [comment]
Now it looks like it stopped working in Chromium too. It was nice while it lasted, but if it's not working on either of the browsers I use forces me to use something else.

**Comment by agrinman** [comment]
Hi everyone -- there's a known issue with GitHub and Firefox, and we haven't been able to address it yet, but everything else should be working (like chromium). There was a issue with push certificates we addressed just recently so please let me know if chromium is still not working...

Just want to reiterate that we're still absolutely supporting Krypton. There's SO much coming, please stay tuned. We can't reply to github issues all the time and there's definitely edge cases we're missing, but we have been updating the public extension versions periodically (though the code releases here are delayed).

**Comment by systemcrash** [comment]
Feels like as long as I've been using FF, Github has been without issue. Oh well. 

Use WebAuthn?

https://blog.mozilla.org/blog/2020/08/20/password-security-part-iv-webauthn/
https://github.blog/2019-08-21-github-supports-webauthn-for-security-keys/

**Comment by systemcrash** [comment]
Still a problem.... ☹️

**Comment by dspaxton** [comment]
RIP krypton. Shame really as it had great use cases
